Debugger/bin/google-cloud-debugger

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env php
+<?php
+/**
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Debugger;
+
+use Google\Cloud\Core\Daemon\DaemonUtils;
```

Comment:
Same comment as `google-cloud-batch` regarding how this class is found.

This script is invoked directly as `vendor/bin/google-cloud-daemon [options]`